app/assets/js/scripts.js

```diff
@@ -182,6 +176,23 @@ app.hookupSteps = function() {
 
 };
 
+app.setPublisher = function($publisher) {
+  app.state.publisher_id = $publisher.data('publisher-id');
+
+  // todo: make eventsArePolygons a property of a publisher
+  app.eventsArePolygons = $publisher.data('publisher-title').match(/Leaf Collection/);
```

Comment:
@bigfleet definitely. The magical matching is a weirdbeard holdover from a polygon proof of concept last year.

I'll put something together for this PR that works more generally.